docs_src/microservices/core/command/GettingStarted.md

```diff
@@ -183,12 +199,52 @@ Example of querying all device core commands via messaging:
 2. Receive query response message from external MQTT broker on topic `edgex/commandquery/response`:
 ```json
 {
-  "ApiVersion":"v2",
-  "ContentType":"application/json",
-  "CorrelationID":"14a42ea6-c394-41c3-8bcd-a29b9f5e6835",
-  "RequestID":"e6e8a2f4-eb14-4649-9e2b-175247911369",
-  "ErrorCode":0,
-  "Payload":"..."
+   "apiVersion":"v3",
```

Comment:
Please check all `apiVersion` values in this file are replaced with `{{api_version}}`.

Reply:
updated